util-core/src/main/java/com/twitter/concurrent/Spools.java

```diff
@@ -22,7 +22,7 @@ private Spools() { }
    * Creates a new `Spool` of given `elems`.
    */
   public static <T> Spool<T> newSpool(Collection<T> elems) {
-    Seq<T> seq = JavaConversions.asScalaBuffer(new ArrayList<T>(elems)).toSeq();
+    Seq<T> seq = JavaConverters.collectionAsScalaIterableConverter(elems).asScala().toSeq();
```

Comment:
I don't *love love* this because of the intermediate object, but we do shave the ArrayList alloc, which is pretty good.  can you add a TODO to change to `collectionAsScalaIterable` when we drop support for 2.11.x?

Reply:
Initially, I mainly tried to make this work with the smallest change possible. Thinking about this more, iterating the java collection right into the spool without any converters might be a better strategy.

If I try to do that, I'm going to struggle a bit, since I don't actually know Java, and my knowledge of the JDK is limited to what I regularly use from Scala.

Reply:
yah, take a stab at it.  java is basically a subset of scala.